content/en/docs/best-practices/channel-naming.md

```diff
@@ -175,9 +202,62 @@ candidate/fast/stable to denote the maturity level of the operator.  Using all
 
 Channel promotion is the notion of moving an operator from one channel to 
 another.  For example, consider the case where you have an operator version 
-1.0.1 which is found in a candidate channel, then you might decide to offer 
+`1.0.1` which is found in a candidate channel, then you might decide to offer 
 support for that version and want to move it to a stable channel.  
 
 Today, channel promotion is achieved by creating a new operator version 
-(1.0.2) that is labeled with the channel(s) you want to promote to (as well 
+(`1.0.2`) that is labeled with the channel(s) you want to promote to (as well 
 as any channels you want to keep it in).
+
+# UPGRADES 
+
+OLM provides a variety of ways to specify updates between operator versions. Before we continue with the following 
+recommendations, make sure you understand the options by checking [Creating an update graph with OLM][upgrade-graph].
+
+## Recommended upgrade path
+
+Within a channel each patch release should be directly upgradable to the [HEAD of channel](/docs/glossary/#channel-head). 
+Use skips or skipRange to provide this behaviour. (i.e. if you followed the above `Example 3` that means 
+use skipRange to publish a patch for `3.6.z`under the`stable-v3.6` would mean setting the skipRange to be 
+`>= 3.5.z < 3.6.z`, where `3.5.z` represents the oldest `3.5.z` version for which 
+you can provide direct upgrade support to `3.6.z` latest)
+
+#### Example
+
+Let's imagine that you will publish the Operator bundle `3.6.30` (patch release under the channel `stable-v3.6`) and
+that you have published so far:
+
+- Operator bundles versions from `3.5.0` to `3.5.25` under the `stable-v3.5` channel
```

Comment:
```suggestion
- Operator bundles versions from `v3.5.0` to `v3.5.25` under the `stable-v3.5` channel
```

This is a nit, for all the versions quoted in the rest of the section too, just to make it look a little more clean by following the semver convention